apps/scorecard/package.json

```diff
@@ -19,5 +19,8 @@
     "object-sizeof": "^1.6.3",
     "popper.js": "^1.16.1",
     "xmlbuilder": "^10.1.1"
+  },
+  "devDependencies": {
+    "vanillajs-datepicker": "^1.2.0"
```

Comment:
This probably shouldn't be a dev dependency. The dev dependencies are usually used for things like test frameworks as they aren't installed in the container image.

Reply:
Actually - looking at `MATScommon/.../package.js` - is this needed here if MATScommon is providing its own copy of vanillajs-datepicker? I'm not sure if packages used by the library are made available to the app that consumes the library.

Reply:
Good catch, it definitely needs to be a client side dependency but not a devDependency. I changed it.